types/tiny-async-pool/index.d.ts

```diff
@@ -22,7 +17,7 @@ export = asyncPool;
  * @template IN Type of the input array
  * @template OUT Type of the resolves of the promises
  */
-declare function asyncPool<IN, OUT>(
+export default function asyncPool<IN, OUT>(
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`.

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Since it does have an actual member named `default`, you can provide this with namespace merging separately as follows:

```ts
function asyncPool(...): ...
declare namespace asyncPool {
    var _unused: typeof asyncPool;
    export {_unused as default};
}

```